### PR TITLE
Maint 3.8

### PR DESCRIPTION
**Pull request by der-christian-wagner**
gr-CyberRadio, libcyberradio, and cyberradiodriver are now updated for gnuradio 3.8 and python3

### CODE REVIEW

File: gr-CyberRadio/CMakeLists.txt [hunk 1]
```diff
@@ -20,99 +21,85 @@
...
-                        VERSION_INFO_MINOR_VERSION VERSION_INFO_MAINT_VERSION)
-    #message(STATUS "UNPACK_VERSION_INFO ${PACKAGE_VERSION} ->")
-    #message(STATUS "* VERSION_INFO_MAJOR_VERSION=${VERSION_INFO_MAJOR_VERSION}")
-    #message(STATUS "* VERSION_INFO_API_COMPAT=${VERSION_INFO_API_COMPAT}")
-    #message(STATUS "* VERSION_INFO_MINOR_VERSION=${VERSION_INFO_MINOR_VERSION}")
-    #message(STATUS "* VERSION_INFO_MAINT_VERSION=${VERSION_INFO_MAINT_VERSION}")
-endif(DEFINED PACKAGE_VERSION)
+set(VERSION_MAJOR 1)
+set(VERSION_API   0)
+set(VERSION_ABI   0)
```
Comment (jwmelto) [inline gr-CyberRadio/CMakeLists.txt:47]: Shouldn't the version be bumped?

File: gr-CyberRadio/cmake/Modules/FindLibCyberRadio.cmake [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+# show results
+MESSAGE("LibCyberRadio Config Results")
+MESSAGE(" * FOUND=${LIBCYBERRADIO_FOUND}")
+MESSAGE(" * INCLUDE_DIR=${LIBCYBERRADIO_INCLUDE_DIR}")
+MESSAGE(" * LIB_DIR=${LIBCYBERRADIO_LIB_DIR}")
+MESSAGE(" * LIB=${LIBCYBERRADIO_LIB}")
+
+MARK_AS_ADVANCED(LIBCYBERRADIO_INCLUDE_DIR LIBCYBERRADIO_LIB_DIR LIBCYBERRADIO_LIB
+                 LIBCYBERRADIO_FOUND)
```
Comment (jwmelto) [inline gr-CyberRadio/cmake/Modules/FindLibCyberRadio.cmake:97]: end the file with a newline?

File: gr-CyberRadio/docs/doxygen/Doxyfile.in.bak [hunk 3]
```diff
@@ -0,0 +1,1910 @@
+# Doxyfile 1.8.4
```
Comment (jwmelto) [inline gr-CyberRadio/docs/doxygen/Doxyfile.in.bak:1]: did you mean to commit this file? (same with the other .bak files)

File: gr-CyberRadio/CMakeLists.txt (hunk 1 above)
Comment (blsmit5728) [reply gr-CyberRadio/CMakeLists.txt:47]: we actually version the package outside of cmake in our internal versioning.  It doesn't get pulled in here. We can edit that.

**Pull request merged**